src/app/cli/src/explore/notebook_server_impl.rs

```diff
@@ -73,20 +73,20 @@ impl NotebookServerImpl {
             "Exposing Notebook server on a host network interface is not yet supported"
         );
 
-        let network_name = "kamu";
+        let network_name = get_random_name_with_prefix("kamu-");
 
         // Delete network if exists from previous run
         let _ = self
```

Comment:
Looks like this is not needed any more - the chances of us finding a network with the same name are extremely low.